components/blitz/src/omero/cmd/mail/SendEmailRequestI.java

```diff
@@ -74,17 +74,13 @@ public SendEmailRequestI(MailUtil mailUtil) {
     public void init(Helper helper) {
         this.helper = helper;
 
-        final EventContext ec = ((LocalAdmin) helper.getServiceFactory()
```

Comment:
Now anyone with an account can use the server to send mail to everyone who has an account with an e-mail address? This seems like a rather dangerous thing to be allowing.

/cc @joshmoore